commands/env.cmd

```diff
@@ -19,6 +19,11 @@ trap '' ERR
 ## define source repository
 if [[ -f "${WARDEN_HOME_DIR}/.env" ]]; then
   eval "$(sed 's/\r$//g' < "${WARDEN_HOME_DIR}/.env" | grep "^WARDEN_")"
+
+  ## Temporary fix to force mutagen activation
+  if [[ ${WARDEN_MUTAGEN_ENABLE} == '' ]]; then
```

Comment:
@joseluisi4 We should follow the previous examples of providing a default for a variable that may be empty.

`WARDEN_MUTAGEN_ENABLE=${WARDEN_MUTAGEN_ENABLE:-1}`